### DISCUSSION TITLE
Help with Backup-DbaDatabase script

### DISCUSSION THREAD

**Discussion by luisflavio** [dataplat/dbatools#7469] [General]
Hi guys,
Could you please help me if I missing anything about generating Backup File Name since the script Backup-DbaDatabase is generating wrong name to the second database.

I'm executing the script below:
Backup-DbaDatabase -SqlInstance SERVER\SQL2014 -Database DB1,DB2 -CopyOnly -Checksum -Verify -ReplaceInName -Path 'C:\Backups\Databases\instancename\dbname\backuptype' -BackupFileName instancename_dbname_backuptype_timestamp.bak -TimeStampFormat yyyyMMdd_HHmmss

The first backup database file is generating with right name (SQL2014_DB_Full_20210625_130931) but the DB2 is generating with wrong name (DB2_20210625_130618) it is missing the instancename.

**Comment by andreasjordan** [comment]
Sorry to say, but "works on my machine". Please update to the latest version of dbatools and try again. If it still fails, you might want to open an issue.
